base/poco/Foundation/include/Poco/Platform.h

```diff
@@ -120,6 +120,8 @@
 #define POCO_ARCH_AARCH64 0x0f
 #define POCO_ARCH_ARM64 0x0f // same as POCO_ARCH_AARCH64
 #define POCO_ARCH_RISCV64 0x10
+#define POCO_ARCH_RISCV32 0x11
```

Comment:
Wondering why constants for RISC-V 32 are defined? That sounds fairly unrelated.

Reply:
I wanted to be consistent with [poco's upstream](https://github.com/pocoproject/poco/blob/devel/Foundation/include/Poco/Platform.h#L138), which doesn't seem necessary right now.